src/SqlPersistence/Saga/SqlDialect_PostgreSql.cs

```diff
@@ -3,16 +3,43 @@
 namespace NServiceBus
 {
     using System.Text;
+    using Logging;
+    using Newtonsoft.Json;
+    using Persistence.Sql;
+    using LogManager = LogManager;
 
     public partial class SqlDialect
     {
         public partial class PostgreSql
         {
+            static ILog log = LogManager.GetLogger<PostgreSql>();
+
             public override string GetSagaTableName(string tablePrefix, string tableSuffix)
             {
                 return $"\"{Schema}\".\"{tablePrefix}{tableSuffix}\"";
             }
 
+            internal override void ValidateJsonSettings(Newtonsoft.Json.JsonSerializer jsonSerializer)
```

Comment:
Is there a reason the serializer is fully qualified here?

Reply:
name clash